### PR TITLE
adds Kafka support

### PR DESCRIPTION
**Pull request by oguzhaneren**

### CODE REVIEW

**Review by aliaksei-goartos — APPROVED** [review]

**Review by Deffiss — APPROVED** [review]

**Comment by Deffiss** [comment]
Looks good, @Hellevar ?

File: src/TestEnvironmenbt.Docker.Containers.Kafka/KafkaContainerWaiter.cs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        {
+            using var adminClient = new AdminClientBuilder(new AdminClientConfig
+                {
+                    BootstrapServers = container.GetUrl(),
+                })
+                .Build();
+            var topicName = $"{container.Name}_topic_health";
+            try
+            {
+                while (true)
```
Comment (Hellevar) [inline src/TestEnvironmenbt.Docker.Containers.Kafka/KafkaContainerWaiter.cs:31]: Please check base class here https://github.com/Deffiss/testenvironment-docker/blob/master/src/TestEnvironment.Docker/BaseContainerWaiter.cs

I'm not sure that you need to perform all these checks in loop by yourself on your level of 'waiting'. Try to simplify and make PerformCheck method as lightweight and fast as possible, like here - https://github.com/Deffiss/testenvironment-docker/blob/master/src/TestEnvironment.Docker.Containers.Postgres/PostgresContainerWaiter.cs

Comment (oguzhaneren) [reply src/TestEnvironmenbt.Docker.Containers.Kafka/KafkaContainerWaiter.cs]: I i simplified waiter

**Review by Hellevar — APPROVED** [review]

**Comment by Deffiss** [comment]
https://www.nuget.org/packages/TestEnvironment.Docker.Containers.Kafka/

**Pull request merged**